content/en/docs/ops/integrations/kyverno-authz-server/index.md

```diff
@@ -0,0 +1,464 @@
+---
+title: Kyverno Authz Server
+description: How to integrate with Kyverno Authz Server.
+weight: 33
+keywords: [integration,kyverno,policy,authorization]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+The [Kyverno Authz Server](https://kyverno.github.io/kyverno-envoy-plugin) is a GRPC authorization server implementing the [Envoy's External Authorization protocol](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/auth/v3/external_auth.proto).
+
+Istio and the Kyverno Authz Server is a solid option to deliver policy quickly and transparently to application team everywhere in the business, while also providing the data the security teams need for audit and compliance.
```

Comment:
Can you give a bit more context for someone who isn't familiar with Kyverno?  Realistically, is this guide for "I use Kyverno and want to bring Istio into my central policy system" or is it for "There is something in Istio I can't do, and so I want to add a third-party thing, and this is going to be my first experience with Kyverno"?